### PR TITLE
Added Cronos/Harmony Support + Extra Changes

### PR DESCRIPTION
**Pull request by Ncookiez**
**Changes Made**
- Added tested and working Multicall2 deployment to Cronos (ID: 25).
- Added tested and working Multicall2 deployment to Harmony (Shard 0 ID: 1666600000).
- Fixed typos in Avalanche enums.
- Removed extraContext syntax from first usage example (example results weren't displaying them anyways).
- Added note on readme regarding needing a Multicall2 deployment (not Multicall) for tryAggregate calls.
- Updated readme with new contract addresses.

### CODE REVIEW

File: src/multicall.ts [hunk 1]
```diff
@@ -539,15 +539,15 @@ export class Multicall {
         return '0x21681750D7ddCB8d1240eD47338dC984f94AF2aC';
       case Networks.arbitrum:
         return '0x80C7DD17B01855a6D2347444a0FCC36136a314de';
-      case Networks.avalauncheFuji:
+      case Networks.avalancheFuji:
```
Comment (joshstevens19) [inline src/multicall.ts:542]: You need to add the case statement for the 2 networks you added!

File: README.md [hunk 2]
```diff
@@ -85,16 +87,12 @@ let provider = ethers.getDefaultProvider();
 // other context like wallet, signer etc all can be passed in as well.
 const multicall = new Multicall({ ethersProvider: provider, tryAggregate: true });
 
-const contractCallContext: ContractCallContext<{extraContext: string, foo4: boolean}>[] = [
+const contractCallContext: ContractCallContext[] = [
```
Comment (joshstevens19) [inline README.md:90]: Keep this example with extra context it just shows how it should work if you want to pass some extra context in (it does work I use it on a few places)

**Review by joshstevens19 — CHANGES_REQUESTED** [review]
Thanks for PR few comments

File: src/multicall.ts (hunk 1 above)
Comment (Ncookiez) [reply src/multicall.ts:542]: Added in d3de741

(I didn't save that file... Whoops.)

File: README.md (hunk 2 above)
Comment (Ncookiez) [reply README.md:90]: I use it as well :)

My logic here is that there is already a section of the readme called `passing extra context to the call`, where this example is shown.

On the section I removed it in, the extra context is added, but the resulting example does not show the extra context that was passed. I don't believe it was intended to add that context when the documentation was written.

Comment (joshstevens19) [reply README.md:90]: Ahhhh ok good spot keep it then thanks for that!

**Review by joshstevens19 — APPROVED** [review]
Thanks!!! I have another deployment I need to get out with this package. I am away tonight but tomorrow il get this deployed for us thanks for the PR

**Pull request merged**